### ISSUE TITLE
Are USB audio interfaces working?

### ISSUE THREAD

**Issue by mmartain** [Mixaill/Ryzentosh-ASUS-X570-PRIMEPRO#1]
I have setup a similar setup: RX5700XT & Ryzen 3950X. Same motherboard.

My RME babyface audio out stutters when using Ableton Live. When using day to day mac, audio is stable. 

Does Ableton Live work correctly on your system?

**Comment by Mixaill** [comment]
Didn't test it with Ableton, but in any way, please try to switch the PCIe slot with RX 5700XT to the 3.0 mode in BIOS.

I had a lot of stutters and crackling (especially in REAPER via ASIO) with Focusrite on every OS (Windows, macOS, Linux) before I switched it to 3.0.

**Comment by mmartain** [comment]
I did have an older bios without those settings, so I updated to newest version, put the bus to GEN 3 and indeed, no more dropouts! That is great! Does this impact gaming performance in any way that you know?

Anyway: thanks for the tip! Worth mentioning on the readme i suppose :)

**Comment by Mixaill** [comment]
Added mention into the readme.

There is no difference in gaming at all. Moreover, for most of the games PCIe gen3 x8 is enough.

I found only two applications where I see any difference:
- AIDA GPGPU memory copy test (which is obviously slowdown because it is limited by PCIe passthrough capability)
- negligible slowdown in Blender Benchmark

>Anyway: thanks for the tip! Worth mentioning on the readme i suppose :)

You are welcome. 

It took me a few weeks in February to figure this out (I even bought an Orico PCIe USB card, that doesn't help).
